plugin/src/main/java/spp/jetbrains/sourcemarker/status/LogStatusBar.java

```diff
@@ -62,12 +63,21 @@
 
     private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("hh:mm:ss a")
             .withZone(ZoneId.systemDefault());
+    private static final String DOLLAR = "$";
+    private static final String EMPTY = "";
```

Comment:
Some of these variables seem like they could go to VariablePattern now. `EMPTY` is being duplicated here.

Reply:
resolved